app/javascript/crayons/Button/Button.jsx

```diff
@@ -94,10 +94,7 @@ export const Button = (props) => {
       {contentType !== 'text' && contentType !== 'icon-right' && Icon && (
         <Icon />
       )}
-      {(contentType === 'text' ||
-        contentType === 'icon-left' ||
-        contentType === 'icon-right') &&
-        children}
+      {children}
```

Comment:
It doesn't have any impact on current usages of the `Button` component, and it makes sense to allow for children to show in all situations so that we can use e.g. invisible spans for labelling or tooltips